docs/faq/help.txt

```diff
@@ -30,23 +49,25 @@ that spammers get caught, but it also means that your first question to the
 list might take a little longer to get answered. We apologize for any
 inconvenience that this policy may cause.
 
-Nobody on `django-users` answered my question! What should I do?
-================================================================
+Nobody answered my question! What should I do?
+==============================================
 
 Try making your question more specific, or provide a better example of your
 problem.
 
-As with most open-source mailing lists, the folks on |django-users| are
-volunteers. If nobody has answered your question, it may be because nobody
-knows the answer, it may be because nobody can understand the question, or it
-may be that everybody that can help is busy. One thing you might try is to ask
-the question on IRC -- visit the `#django IRC channel`_ on the Freenode IRC
-network.
+As with most open-source projects, the folks on these channels are volunteers.
+If nobody has answered your question, it may be because nobody knows the
+answer, it may be because nobody can understand the question, or it may be that
+everybody that can help is busy.
+
+You can also try asking on a different channel. But please don't post your
+question in all three channels in quick succession.
 
-You might notice we have a second mailing list, called |django-developers| --
-but please don't email support questions to this mailing list. This list is
-for discussion of the development of Django itself. Asking a tech support
-question there is considered quite impolite.
+You might notice we have a second mailing list, called |django-developers|.
+Please don't email support questions to this mailing list. This list is for
+discussion of the development of Django itself. Asking a tech support question
+there is considered impolite, and you will likely simply be directed to ask on
```

Comment:
I think the use of 'impolite' is a little strong, and I think the ordering could be switched around a bit. Let's tell people what it's for and then say why tech support questions shouldn't be asked here. 

Here is my attempt. 

```
You might notice we have a second mailing list, called |django-developers|.
This list is for discussion of the development of Django itself. Please don't 
email support questions to this mailing list as you are likely to be 
directed to ask on |django-users|.
```

Reply:
Yes good point. I agree we should explain first.

I'm not sure "impolite" is that strong. A strong word would be "rude". Removing any such description feels too weak to me as it implies the only negative thing is that you'll be redirected.

Even with the current wording we still get maybe 1-5 posts a week on there asking support questions that I turn down with an auto-fill message.

Reply:
We could go all _Captain Hook_ and have _is considered bad form..._ (🙂 Not necessarily 100% serious... — but maybe.)

I keep tripping on the _"likely simply"_ — can we find a better phrasing for that?

Reply:
I simply dropped "simply"